docs/how-tos/upload-data.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "How to Upload a Batch of Data"
+linkTitle: "Upload a batch of data"
+description: "Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app."
+weight: 100
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+image: "/services/icons/data-capture.svg"
+imageAlt: "Upload data to the Viam app"
+images: ["/services/icons/data-capture.svg"]
+aliases:
+  - /data/upload/
+  - /services/data/upload/
+---
+
+If you configured [data capture](/services/data/capture/) on your machine, Viam automatically uploads data to the cloud from the directory you specified, at the interval you specified.
+However, if you want to upload a batch of data once from somewhere else, either from a different directory on your machine or from your personal computer or mobile device, you have several options:
+
+- Configure the path to your directory in the Viam app, wait for the files to sync, then delete the path.
+  This option requires that you have `viam-server` installed on the machine.
+- Run a Python script to upload files from a folder using the data client API `file_upload_from_path` method.
+  You can do this on a computer that doesn't have `viam-server` installed on it.
+- Upload images from your mobile device using the Viam mobile app.
+
+{{% alert title="In this page" color="tip" %}}
+
+- [Sync a batch of data from another directory](#sync-a-batch-of-data-from-another-directory)
+- [Upload data with the Python SDK](#upload-data-with-python)
+- [Upload images with the Viam mobile app](#upload-images-with-the-viam-mobile-app)
+
+{{% /alert %}}
```

Comment:
I think this is duplicated and doesn't need to be. You can tweak it a bit but I think this keeps it more concise.

```suggestion
However, if you want to upload a batch of data once from somewhere else, either from a different directory on your machine or from your personal computer or mobile device, you have several options using the Viam app, the data client API, or the Viam mobile app.

{{% alert title="In this page" color="tip" %}}

- [Sync a batch of data from another directory](#sync-a-batch-of-data-from-another-directory) by configuring the path to the directory as an additional sync path in a machine's data management service. This requires the data to be on a machine running `viam-server`.
- [Upload data with the Python SDK](#upload-data-with-python) by running a Python script to upload files from a folder.
- [Upload images with the Viam mobile app](#upload-images-with-the-viam-mobile-app) from your mobile device.

{{% /alert %}}
```

Reply:
I thought the same thing but feared doing it in this long form way would be deemed to be inconsistent with the other pages :D Since you agree I'm happy to make this change